app/serializers/user.js

```diff
@@ -1,5 +1,15 @@
 import DS from 'ember-data';
 
 export default DS.RESTSerializer.extend({
-  primaryKey: 'username',
+  attrs: {
```

Comment:
These properties aren't accepted by the API endpoint so no point in sending them